dockerfile_parse/parser.py

```diff
@@ -443,26 +442,24 @@ def _modify_instruction_label_env(self, instruction, instr_key, instr_value):
                     endline = candidate['endline']
                     break
             else:  # LABEL/ENV "name"="value"
-                for index, token in enumerate(splits):
-                    words = token.split("=", 1)
-                    if words[0] == instr_key:
+                for index, token in enumerate(words):
+                    key_val = token.split("=", 1)
```

Comment:
Since we're asserting that length is 2, what about using: `key, val = token.split("=", 1)`

Reply:
Good idea. Added another commit to do that, which I'll squash before merging.